src/app/modules/main/browser_section/dapps/dapps.nim

```diff
@@ -3,11 +3,12 @@ import NimQml, Tables, strutils, strformat
 type
   ModelRole {.pure.} = enum
     Name = UserRole + 1
+    Address
 
 QtObject:
   type
     DappsModel* = ref object of QAbstractListModel
-      items: seq[string]
+      items: seq[(string, string)]
```

Comment:
What do you think of using a named tuple here instead of it being anonymous? should improve readability compared to using indexes

Reply:
Directly switch to object!